### PR TITLE
DTSCCI-349 Remove code smell PR2

### PR DESCRIPTION
**Pull request by tiagreaney**
### JIRA link (if applicable) ###

https://tools.hmcts.net/jira/browse/DTSCCI-349

### Change description ###

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ ] No
```

### CODE REVIEW

File: src/main/java/uk/gov/hmcts/reform/civil/handler/callback/user/JudgmentPaidInFullCallbackHandler.java [hunk 1]
```diff
@@ -56,11 +55,11 @@ private CallbackResponse buildConfirmation(CallbackParams callbackParams) {
     }
 
     private String getHeader() {
-        return format("# Judgment marked as paid in full");
+        return "# Judgment marked as paid in full";
```
Comment (ruban72) [inline src/main/java/uk/gov/hmcts/reform/civil/handler/callback/user/JudgmentPaidInFullCallbackHandler.java:58]: Should these be changed to String constants?

File: src/main/java/uk/gov/hmcts/reform/civil/handler/callback/user/ReferToJudgeDefenceReceivedCallbackHandler.java [hunk 2]
```diff
@@ -47,11 +46,11 @@ private CallbackResponse buildConfirmation(CallbackParams callbackParams) {
     }
 
     private String getHeader() {
-        return format("# The case has been referred to a judge for a decision");
+        return "# The case has been referred to a judge for a decision";
```
Comment (ruban72) [inline src/main/java/uk/gov/hmcts/reform/civil/handler/callback/user/ReferToJudgeDefenceReceivedCallbackHandler.java:49]: same as above

**Review by ruban72 — APPROVED** [review]

File: src/main/java/uk/gov/hmcts/reform/civil/handler/callback/user/StandardDirectionOrderDJ.java [hunk 3]
```diff
@@ -911,39 +909,6 @@ private DynamicList deleteLocationList(DynamicList list) {
...
-    }
-
-    private Boolean checkLocation(final LocationRefData location, String locationTempLabel) {
-        String locationLabel = location.getSiteName()
-            + " - " + location.getCourtAddress()
-            + " - " + location.getPostcode();
-        return locationLabel.equals(locationTempLabel);
-    }
-
-    private DynamicList getLocationListFromCaseData(DynamicList hearingList, DynamicList trialList) {
```
Comment (velimd) [inline src/main/java/uk/gov/hmcts/reform/civil/handler/callback/user/StandardDirectionOrderDJ.java:937]: are these functions being moved somewhere or were they removed as they were unused?

File: src/main/java/uk/gov/hmcts/reform/civil/helpers/LocationHelper.java [hunk 4]
```diff
@@ -183,7 +182,6 @@ private Optional<RequestedCourt> getSpecClaimantRequestedCourt(CaseData caseData
     private Optional<RequestedCourt> getUnspecClaimantRequestedCourt(CaseData caseData) {
         return Optional.ofNullable(caseData.getCourtLocation())
             .map(courtLocation -> RequestedCourt.builder()
-                .requestHearingAtSpecificCourt(YesOrNo.YES)
```
Comment (velimd) [inline src/main/java/uk/gov/hmcts/reform/civil/helpers/LocationHelper.java:186]: was this removed for a reason?

File: src/main/java/uk/gov/hmcts/reform/civil/helpers/judgmentsonline/JudgmentOnlineMapper.java [hunk 5]
```diff
@@ -57,8 +57,8 @@ private Integer getNextJudgmentId(CaseData caseData) {
     }
 
     private boolean isHistoricJudgment(JudgmentDetails activeJudgment) {
-        return JudgmentState.CANCELLED.equals(activeJudgment.getState())
-            || JudgmentState.SET_ASIDE_ERROR.equals(activeJudgment.getState())
-            || JudgmentState.SET_ASIDE.equals(activeJudgment.getState()) ? true : false;
+        return uk.gov.hmcts.reform.civil.model.judgmentonline.JudgmentState.CANCELLED.equals(activeJudgment.getState())
+            || uk.gov.hmcts.reform.civil.model.judgmentonline.JudgmentState.SET_ASIDE_ERROR.equals(activeJudgment.getState())
+            || uk.gov.hmcts.reform.civil.model.judgmentonline.JudgmentState.SET_ASIDE.equals(activeJudgment.getState());
```
Comment (velimd) [inline src/main/java/uk/gov/hmcts/reform/civil/helpers/judgmentsonline/JudgmentOnlineMapper.java:62]: i feel like it was cleaner before, can we not just import JudgmentState?

File: src/main/java/uk/gov/hmcts/reform/civil/model/docmosis/settlementagreement/SettlementAgreementFormMapper.java [hunk 6]
```diff
@@ -43,10 +43,6 @@ public SettlementAgreementForm buildFormData(CaseData caseData) {
                 getContactPerson(defendantDetails)
         );
 
-        String totalClaimAmount = Optional.ofNullable(caseData.getTotalClaimAmount())
```
Comment (velimd) [inline src/main/java/uk/gov/hmcts/reform/civil/model/docmosis/settlementagreement/SettlementAgreementFormMapper.java:46]: reason this was removed?

File: src/main/java/uk/gov/hmcts/reform/civil/service/docmosis/dj/DefaultJudgmentOrderFormGenerator.java [hunk 7]
```diff
@@ -295,11 +295,6 @@ static String fillTypeBundleInfo(CaseData caseData) {
         return "";
     }
 
-    private String fillTypeBundleInfoTrial() {
```
Comment (velimd) [inline src/main/java/uk/gov/hmcts/reform/civil/service/docmosis/dj/DefaultJudgmentOrderFormGenerator.java:298]: unused?

File: src/main/java/uk/gov/hmcts/reform/civil/handler/callback/user/StandardDirectionOrderDJ.java (hunk 3 above)
Comment (tiagreaney) [reply src/main/java/uk/gov/hmcts/reform/civil/handler/callback/user/StandardDirectionOrderDJ.java:937]: yes fillPreferredLocationData and getLocationListFromCaseData are unused methods

File: src/main/java/uk/gov/hmcts/reform/civil/helpers/LocationHelper.java (hunk 4 above)
Comment (tiagreaney) [reply src/main/java/uk/gov/hmcts/reform/civil/helpers/LocationHelper.java:186]: yes requestHearingAtSpecificCourt is deprecated

File: src/main/java/uk/gov/hmcts/reform/civil/helpers/judgmentsonline/JudgmentOnlineMapper.java (hunk 5 above)
Comment (tiagreaney) [reply src/main/java/uk/gov/hmcts/reform/civil/helpers/judgmentsonline/JudgmentOnlineMapper.java:62]: yeah will update this to import JudgmentState

File: src/main/java/uk/gov/hmcts/reform/civil/model/docmosis/settlementagreement/SettlementAgreementFormMapper.java (hunk 6 above)
Comment (tiagreaney) [reply src/main/java/uk/gov/hmcts/reform/civil/model/docmosis/settlementagreement/SettlementAgreementFormMapper.java:46]: Unused local variable

File: src/main/java/uk/gov/hmcts/reform/civil/service/docmosis/dj/DefaultJudgmentOrderFormGenerator.java (hunk 7 above)
Comment (tiagreaney) [reply src/main/java/uk/gov/hmcts/reform/civil/service/docmosis/dj/DefaultJudgmentOrderFormGenerator.java:298]: Yes unused

**Review by velimd — DISMISSED** [review]

**Review by ruban72 — APPROVED** [review]

**Review by rishikrsharma — APPROVED** [review]

**Pull request merged**